contracts/auth/helpers/BaseAuthAddresses.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.10;
+
+contract BaseAuthAddresses {
+    address internal constant ADMIN_VAULT_ADDR = 0xD8E67968d8a0df4beCf2D50daE1e34d4d80C701C;
+    address internal constant FACTORY_ADDRESS = 0x7783da8958013a57a5514737a4FBDFF06A0056e1;
```

Comment:
I'd maybe rename this to DSGUARD_FACTORY_ADDRESS (everywhere) so no confusion can happen between dsproxy factory and dsguard factory